sql/core/src/test/resources/sql-tests/inputs/aliases.sql

```diff
@@ -0,0 +1,17 @@
+-- Test data.
```

Comment:
How about we name this file `table-aliases.sql`; that seems a little bit less confusing.